### PR TITLE
Respect exit code for executed binaries

### PR DESCRIPTION
**Pull request by fastner**
This PR checks all exit codes of executed binaries and if one of it is `> 0` preppy itself exits with an exit code of `1`.

### CODE REVIEW

**Comment by swernerx** [comment]
Thanks for the PR. 

Seems like the folder name of the test got a typo. "fialing"

File: src/getRollupInputOptions.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ import cjsPlugin from "rollup-plugin-commonjs"
 import executablePlugin from "rollup-plugin-executable"
 import figures from "figures"
 import jsonPlugin from "rollup-plugin-json"
+import jsx from "acorn-jsx"
 import rebasePlugin from "rollup-plugin-rebase"
 import replacePlugin from "rollup-plugin-replace"
-import runPlugin from "rollup-plugin-run"
 import { terser as terserPlugin } from "rollup-plugin-terser"
-import jsx from "acorn-jsx"
+import jsxPlugin from "./jsxPlugin"
```
Comment (swernerx) [inline src/getRollupInputOptions.js]: Not sure why these were changed in order... seems unrelated

**Comment by swernerx** [comment]
Thinking about watch mode, we have to figure out what the user assumption (=developer assumption might be here). It feels sensible to allow errors during this and just wait for changes to be made (with reporting the issues for sure).

File: src/getRollupInputOptions.js (hunk 1 above)
Comment (fastner) [reply src/getRollupInputOptions.js]: Don't know. Something eslint/prettier has done.

**Comment by fastner** [comment]
In watch mode errors are written to the terminal but preppy is not stopped.

**Comment by fastner** [comment]
As developer is aborting preppy via CTRL-C or something like that in `watch mode` I don't think she is expecting a valid exit code here. But I can figure out if it is possible to set exit code in this case. The CLI has the last execution state of the script, so if it fails 2 times and then works correctly it has an exit code of zero.

File: src/getRollupInputOptions.js (hunk 1 above)
Comment (fastner) [reply src/getRollupInputOptions.js]: I have reverted these changes.

**Comment by swernerx** [comment]
Looks good! Thanks!

**Comment by swernerx** [comment]
Released as v8.2.0
